cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/DefaultSecureStoreServiceTest.java

```diff
@@ -88,21 +79,16 @@
   public static void setup() throws Exception {
     SConfiguration sConf = SConfiguration.create();
     sConf.set(Constants.Security.Store.FILE_PASSWORD, "secret");
-    final Injector injector = Guice.createInjector(new AppFabricTestModule(createCConf(), sConf));
-    injector.getInstance(TransactionManager.class).startAndWait();
-    injector.getInstance(DatasetOpExecutor.class).startAndWait();
-    discoveryServiceClient = injector.getInstance(DiscoveryServiceClient.class);
-    DatasetService datasetService = injector.getInstance(DatasetService.class);
-    datasetService.startAndWait();
-    waitForService(Constants.Service.DATASET_EXECUTOR);
+    final Injector injector = AppFabricTestHelper.getInjector(createCConf(), sConf, new AbstractModule() {
+      @Override
```

Comment:
We only have a single test that needs ``SConfiguration`` and overrides. So I did not want to expose it in ``AppFabricTestHelper``